lib/index.js

```diff
@@ -343,3 +372,13 @@ function _updateSessionAuthenticationData(
   }
   return data;
 }
+
+function _toBase64NoPad(x) {
+  return x.toString('base64', 0, 16).split('=')[0];
+}
+
+function _sleep({delay}) {
+  return new Promise(resolve => {
+    setTimeout(resolve, delay);
+  });
```

Comment:
```js
// with the other imports
import {setTimeout} from 'node:timers/promises'; 
```

```suggestion
async function _sleep({delay}) {
  return setTimeout(delay);
```

Reply:
I'd just remove the _sleep wrapper and call this directly.